website/docs/components/pagination/partials/accessibility/accessibility.md

```diff
@@ -17,13 +17,13 @@ Focus and move between pagination controls.
 
 `tab`
 
-![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-01.png =50%x50%)
+![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-01.png =836x*)
```

Comment:
Try to remove the sizes and see what happens :)
If the size of the image is (at double resolution) the same as the width of the container, it's automatically scaled and so there's no need of explicitly setting a size (which is extra work).
That was the "core" of my suggestion about images in the documentation ;)

```suggestion
![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-01.png)
```